oak-doc/src/site/markdown/query/elastic.md

```diff
@@ -0,0 +1,47 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+  -->
+
+## Elastic Index
+
+Oak supports Elasticsearch (Elastic for short) based indexes to support 
+both property constraint and full text constraints. 
+Elastic indexes support similar features than [Lucene][lucene] indexes, 
+however there are differences:
+
+* The `type` is `elastic`.
+* The index definition needs to be under `/oak:index`.
+  Other locations are not supported.
+* The `async` property needs to be set to `async`. 
```

Comment:
The `async` property needs to be set to `async`.  --> The `async` property needs to be set to `elastic-async`.